applogic/Cargo.toml

```diff
@@ -33,7 +33,7 @@ anyhow = { version = "1", features = ["backtrace"] }
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 tokio = { version = "1.39", features = ["rt", "macros"] }
-flutter_rust_bridge = { version = "=2.7.0", features = ["chrono", "uuid"] }
+flutter_rust_bridge = { version = "=2.7.1", features = ["chrono", "uuid"] }
```

Comment:
Nitpicking, but let's not make code changes in a `docs:` PR.

Reply:
I believe main was merged into this branch as that what’s the GitHub UI recommends to make the PR mergeable. Not my change but upstream.

Reply:
Yeah, it’s the latter. Simply running the installation steps may result in changes being made. Not an intentional commit and I somehow missed excluding those changes. May want to look into that since that’s likely to repeat in the future if you have other contributors.